zio-http-gen/src/test/scala/zio/http/gen/openapi/EndpointGenSpec.scala

```diff
@@ -594,6 +594,24 @@ object EndpointGenSpec extends ZIOSpecDefault {
           )
           assertTrue(scala.files.head == expected)
         },
+        test("response are case class with 22+ fields") {
```

Comment:
You need to add a test to the OpenAPIGenSepc. You can keep this spec, but please let it look like the other specs